types.d.ts

```diff
@@ -40,6 +57,8 @@ declare module 'mdast' {
   interface BlockContentMap {
     [NodeTypes.callout]: Callout;
     [NodeTypes.codeTabs]: CodeTabs;
+    [NodeTypes.embed]: Embed;
```

Comment:
I believe you want to add this to the `RootContentMap` too. I'm guessing it's probably the case that the link that the embed becomes is always wrapped in a paragraph, but that seems incorrect.

Reply:
ok cool, i wasn't 100% sure